flocker/ca/_ca.py

```diff
@@ -95,10 +99,10 @@ def create_certificate_authority(keypair, dn, request, serial, expiry, digest):
 
     :param int serial: The certificate serial number.
 
-    :param int expiry: Number of seconds from now until this certificate
-        should expire.
+    :param bytes expiry: ASN1 formatted datetime string representing the
```

Comment:
For future reference, a nice Python-y API would probably just accept a `datetime.datetime` here and do the string formatting itself.  We don't care that much about this as an API, though.